test/Microsoft.Health.Fhir.Tests.E2E/Common/KnownResourceTypes.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Common
+{
+    public static class KnownResourceTypes
```

Comment:
Would this be useful in ValueSets or Core? I know @feordin was using a known type